docker/env/version

```diff
@@ -1 +1 @@
-0.29
+0.31
```

Comment:
why do we skip 0.30?

Reply:
Isreal noticed me that 30 could be in another PR. So it is just to be sure that it is not repeated.